@navikt/aksel/src/codemod/transforms/darkside/box-to-boxnew/box-to-boxnew.ts

```diff
@@ -0,0 +1,57 @@
+import type { API, FileInfo } from "jscodeshift";
+import { legacyTokenConfig } from "../../../../darkside/config/legacy.tokens";
+import {
+  findComponentImport,
+  findJSXElement,
+  findProp,
+} from "../../../utils/ast";
+import { getLineTerminator } from "../../../utils/lineterminator";
+
+const propsAffected = ["background", "borderColor", "shadow"];
+
+export default function transformer(file: FileInfo, api: API) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+
+  const toSourceOptions = getLineTerminator(file.source);
+
+  const sourceName = findComponentImport({
+    file,
+    j,
+    name: "Box",
+    packageType: "react",
+  });
+
+  if (!sourceName) {
+    return;
+  }
+
+  const astElements = findJSXElement({
+    root,
+    j,
+    name: sourceName,
+    originalName: "Box",
+  });
+
+  for (const astElement of astElements.paths()) {
+    for (const prop of propsAffected) {
+      findProp({ j, path: astElement, name: prop }).forEach((attr) => {
+        const attrvalue = attr.value.value;
+        if (attrvalue.type === "StringLiteral") {
+          const config = legacyTokenConfig[attrvalue.value];
+          if (config?.replacement) {
+            attrvalue.value = config.replacement;
+          }
+          // else {
```

Comment:
Yea would be nice if we could insert a comment above the line, or just simply at the top of the file with a spesific keyword, and potentially what they need to change 🥇